systems/andes_user_guide.rst

```diff
@@ -1128,35 +1128,42 @@ please contact the OLCF User Assistance Center.
 Debugging
 =========
 
-Arm DDT
+Linaro DDT
 -------
 
```

Comment:
Underline to short for section title

Reply:
fixed in 15e4ea35ac8e6c884ee1e03be22a3c76c1d214d3